autoload/dictionary.vim

```diff
@@ -10,7 +10,8 @@ vim9script
 export var options: dict<any> = {
     enable: false,
     icase: false,
-    maxCount: 10,
+    maxWords: 8000,
```

Comment:
Why did you add `maxWords` when `maxCount` is doing the same? Also, your screen can display <100 lines. Why are you searching for 8000 words? Remember, it searches dictionary every time you type a character.